scripts/release/packager/create-mapbox-maps-xcframework.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+# Simulator
+step "Archiving iPhone simulator framework"
+xcodebuild  -archivePath 'pkg/MapboxMaps-simulator.xcarchive' \
+		    -project 'MapboxMapsPackager.xcodeproj' \
+		    -scheme MapboxMaps \
+		    -sdk iphonesimulator \
+		    -configuration Release \
+		    -derivedDataPath .derivedDataPath-simulator \
+			archive \
+			ARCH=x86_64 \
+			ONLY_ACTIVE_ARCH=YES \
```

Comment:
Let's let it build for all supported architectures. This will help pave the way for Apple Silicon